walk-safe/package.json

```diff
@@ -3,12 +3,20 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@apollo/client": "^3.3.21",
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+<<<<<<< HEAD
+    "apollo-server": "^2.25.2",
+    "graphql": "^15.5.1",
+=======
+>>>>>>> ba6f0d65abd8c2663809ad540d256f37b30d78a9
     "node-sass": "^6.0.1",
```

Comment:
Nice, thx for getting node-sass back in our dependencies.  Hopefully this time it sticks